cypress-tests/integration-tests/features/autocomplete.feature

```diff
@@ -4,24 +4,49 @@ Feature: Autocomplete
     Given I navigate to the handler page
     And I click the Show Filter button
 
-  Scenario: Autocomplete filter is displayed
-    Then the Autocomplete box is shown
+  Scenario: Static autocomplete filter is displayed
+    Then the static Autocomplete box is shown
 
-  Scenario: Autocomplete options are displayed
-    When I enter text longer than the minimum data length
-    Then a list of matching options is displayed beneath the autocomplete box
+  Scenario: Static autocomplete options are displayed
+    When I enter text longer than the minimum data length into the static Autocomplete box
```

Comment:
Maybe it would be good to add a scenario also for a case when the text length is exactly equal to the minimum data length. 
What would be the expected behaviour then?

Reply:
Accidentally, this is what it's currently testing. 😄 

"shorter" is 2, "longer" is 3, and the minLength is 3.

Reply:
I just noticed it in the code below/